Packs/Censys/Integrations/CensysV2/CensysV2_test.py

```diff
@@ -0,0 +1,167 @@
+
+import json
+import io
+import demistomock as demisto
+import CensysV2
+from CensysV2 import Client, censys_view_command, censys_search_command, main
+import pytest
+
+SEARCH_HOST_OUTPUTS = [{
+    'ip': '1.0.0.0',
+    'services': [
+        {'port': 80, 'service_name': 'HTTP', 'transport_protocol': 'TCP'},
+        {'port': 443, 'service_name': 'HTTP', 'transport_protocol': 'TCP'}],
+    'location': {'continent': 'Oceania', 'country': 'Australia', 'country_code': 'AU', 'timezone': 'Australia/Sydney',
+                 'coordinates': {'latitude': -33.494, 'longitude': 143.2104}, 'registered_country': 'Australia',
+                 'registered_country_code': 'AU'},
+    'autonomous_system': {'asn': 13335, 'description': 'CLOUDFLARENET', 'bgp_prefix': '1.0.0.0/24',
+                          'name': 'CLOUDFLARENET', 'country_code': 'US'}}]
+
+
+SEARCH_CERTS_OUTPUTS = [{
+    'parsed':
+        {'fingerprint_sha256': 'f3ade17dffcadd9532aeb2514f10d66e22941393725aa65366ac286df9b41234',
+         'issuer': {'organization': ["Let's Encrypt"]},
+         'issuer_dn': "C=US, O=Let's Encrypt, CN=Let's Encrypt Authority X3",
+         'names': ['*.45g4rg43g4fr3434g.gb.net', '45g4rg43g4fr3434g.gb.net'],
+         'subject_dn': 'CN=45g4rg43g4fr3434g.gb.net',
+         'validity': {'end': '2021-01-10T14:46:11Z', 'start': '2020-10-12T14:46:11Z'}}}]
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+@pytest.fixture()
+def client():
+    client = Client(base_url='https://search.censys.io/api/', auth=('test', '1234'), verify=True, proxy=False)
+    return client
+
+
+def test_censys_host_search(mocker, client):
+    """
+    Given:
+        Command arguments: query and limit
+    When:
+        Running cen_search_command
+    Then:
+        Validate the output compared to the mock output
+    """
+    args = {
+        'index': 'ipv4',
+        'query': 'services.service_name:HTTP',
+        'limit': 1
+    }
+
+    mock_response = util_load_json('test_data/search_host_response.json')
+    mocker.patch.object(client, 'censys_search_ip_request', return_value=mock_response)
```

Comment:
why are there a few places that we still use `mocker` instead of `requests_mock` ?